src/main/frontend/pages/TeamFormationStack.tsx

```
@@ -0,0 +1,485 @@
+    return (
+      <Card title="🗳️ 투표 기반 팀 분배 설정">
+        <div className="space-y-4">
+          <Input 
+            label="투표 질문 (예: 함께하고 싶은 사람, 선호하는 역할)" 
+            value={pollQuestion} 
+            onChange={(e) => setPollQuestion(e.target.value)} 
+          />
+          <div>
+            <label className="block text-sm font-medium text-neutral-700 mb-1">투표 항목:</label>
+            {pollOptions.map((option, index) => (
+              <div key={index} className="flex items-center space-x-2 mb-2">
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)